linode/manager
# upcoming: [M3-7866] - Improve Proxy Account Visibility with Distinct Visual Indicators

## Description 📝
When navigating through accounts, it's essential to quickly identify proxy accounts for clarity and efficient management. To enhance this distinction, we propose implementing visual cues such as adding a border around the profile picture and altering the user icon. This enhancement will ensure proxy accounts are visually differentiated.

## Changes  🔄
- Proxy users should not be able to update their gravatar.

## Target release date 🗓️
3/18

## Preview 📷
| Light  | Dark   |
| ------- | ------- |
| ![Screenshot 2024-03-12 at 4 37 53 PM](https://github.com/linode/manager/assets/125309814/22b88643-27b2-4b41-9c97-8995c7e77884) | ![Screenshot 2024-03-12 at 4 38 01 PM](https://github.com/linode/manager/assets/125309814/5b08e22c-5a2c-4375-ab71-969a98f20a0e) |

## How to test 🧪

### Prerequisites
- Log into Parent account using dev creds
- Switch into a child account

### Verification steps
- Observe that the proxy account has the new visual indicators
- Go to http://localhost:3000/profile/display and observe that there is no Gravatar profile photo section, so that proxy users cannot set gravatars

## As an Author I have considered 🤔

*Check all that apply*
